package.json

```diff
@@ -45,8 +45,8 @@
     "new:component": "hygen component new",
     "new:type": "hygen type new",
     "new:util": "hygen util new",
-    "new:slice": "hygen slice new && eslint -c ./.eslintrc.js 'src/store/**/*.{ts,tsx}'",
-    "new:store": "hygen store new && eslint -c ./.eslintrc.js 'src/store/*.{ts,tsx}'",
+    "new:slice": "hygen slice new && eslint 'src/store/**/*.{ts,tsx}' --fix",
+    "new:store": "hygen store new && eslint 'src/store/*.{ts,tsx}' --fix",
```

Comment:
These could probably use `run-s` from `npm-run-all` so that they are compatible on windows

Reply:
Do you have an example how to add `run-s` to this?  I tried and kept getting issues.

Reply:
You would probably have to split the two steps into their own scripts then run them both with `run-s`. Something like:
```
"new:slice": "run-s new:slice:*",
"new:slice:run": "hygen slice new",
"new:slice:lint": "eslint 'src/store/**/*.{ts,tsx}' --fix",
```

Reply:
I wonder if it would be better to just manually run this script in the `sh:` part of the template file—I think so. Then you have immediate context to the path etc. 🤷‍♂

Reply:
But also, I love running the formatter on all these. Definitely going to add that to all the generators 🎉